### PR TITLE
add dkeyserver and dkeycache docker support

### PR DESCRIPTION
**Pull request by wanghouqi**
Signed-off-by: wanghouqi <wanghouqi@vip.qq.com>

### CODE REVIEW

File: docker/dkeyserver/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    libsgx-quote-ex-dev  \
+    libsgx-uae-service \
+    libsgx-ra-network \
+    libsgx-ra-uefi \
+    libsgx-dcap-ql-dev 
+
+
+# RESET PCCS URL
+WORKDIR /etc
+RUN echo '# cat sgx_default_qcnl.conf' > sgx_default_qcnl.conf \
```
Comment (yang8621) [inline docker/dkeyserver/Dockerfile:62]: remove

File: core/App/fifo.cpp [hunk 2]
```diff
@@ -47,7 +47,7 @@
 
 #define BUFFER_SIZE 1024
 
-#define UNIX_DOMAIN "/tmp/UNIX.domain"
+#define UNIX_DOMAIN "/var/run/dkey/dkeyprivision.sock"
```
Comment (syan10) [inline core/App/fifo.cpp:50]: "/var/run/dkey/dkeyprivision.sock"  --> "/var/run/ehsm/dkeyprovision.sock"

Comment (wanghouqi) [reply core/App/fifo.cpp]: It will be change

File: docker/dkeyserver/Dockerfile (hunk 1 above)
Comment (wanghouqi) [reply docker/dkeyserver/Dockerfile]: It will be remove

**Review by yang8621 — APPROVED** [review]

File: core/App/fifo.cpp [hunk 3]
```diff
@@ -47,7 +47,7 @@
 
 #define BUFFER_SIZE 1024
```
Comment (syan10) [inline core/App/fifo.cpp:48]: A typo in the commit title
"add dkeyserver and ckeycache docker support" --> "add dkeyserver and dkeycache docker support"

Comment (wanghouqi) [reply core/App/fifo.cpp:48]: It will be fix

**Review by yang8621 — APPROVED** [review]

**Review by syan10 — APPROVED** [review]

**Pull request merged**